### ISSUE TITLE
Publish docker image to container registry

### ISSUE THREAD

**Issue by obbardc** [juriansluiman/groceri.es#22]
Should publish a docker container to ghcr.io or dockerhub using GitHub actions.

**Comment by juriansluiman** [comment]
Yeah I'm kinda new to this type of development, but this should be pushed as an app to some sort of registry 👍

**Comment by obbardc** [comment]
this is the main thing blocking me from deploying groceri.es, if you enable these in GitHub in preparation i will try to spare some time for an initial implementation:
- GitHub container registry
- GitHub actions

**Comment by obbardc** [comment]
Fixed in #24
